spec.go

```diff
@@ -18,11 +18,16 @@ import (
 	sdk "github.com/conduitio/conduit-connector-sdk"
 )
 
+// version is set during the build process (i.e. the Makefile).
+// It follows Go's convention for module version, where the version
+// starts with the letter v, followed by a semantic version.
+var version = "dev"
+
 func Specification() sdk.Specification {
 	return sdk.Specification{
 		Name:    "postgres",
 		Summary: "A PostgreSQL source and destination plugin for Conduit.",
-		Version: "v0.1.0",
+		Version: version,
```

Comment:
I just noticed that the Kafka PR is failing because of the version check in the acceptance tests (doesn't start with v).